apps/partner2/stylesheets/overview.styl

```diff
@@ -28,7 +28,7 @@
       min-height 100px
       .partner-overview-section-title
         avant-garde-size('m-headline')
-    &[data-module=about] + [data-module=news]
+    &[data-module=news]
```

Comment:
In the [initial implementation](https://github.com/artsy/force-private/pull/4273), when there is no About section, we want to display the News section horizontally, as @brentcas [suggested](https://artsy.slack.com/archives/G3D737CJ2/p1481236437000017), and hide the section title. That's why we have the adjacent sibling selector here. Looks like the styles got messed up somehow afterwards. Let's maybe merge this for the time being so it doesn't look broken, and follow up with another PR and revisit.

![image](https://cloud.githubusercontent.com/assets/796573/21031629/7443810e-bd72-11e6-870a-2bbe54ac2b69.png)